app/assets/javascripts/shop/session.js

```diff
@@ -0,0 +1,22 @@
+document.addEventListener('turbolinks:load', function() {
+  $('.modal').on('shown.bs.modal', function () {
+    correctPhoneNumber();
+  });
+  correctPhoneNumber();
```

Comment:
Vì cái log in signup có 2 nơi anh ạ. 1 cái modal, 1 cái view thường. nên phải gọi 2 lần nó mới đầy đủ dc cả 2 ạ